### PR TITLE
Added Hugging Face tools where applicable

### PR DESCRIPTION
**Pull request by joshchagani**
- Included a Hugging Face blog tutorial on how to train a new language model from scratch. 
- Created a new "Libraries and Tooling" section to allow more NLP contributors and added Hugging Face tools.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -237,6 +239,11 @@ NLP as API with higher level functionality such as NER, Topic tagging and so on
 - [rstWeb](https://corpling.uis.georgetown.edu/rstweb/info/) - open source local or online tool for discourse tree annotations
 - [GitDox](https://corpling.uis.georgetown.edu/gitdox/) - open source server annotation tool with GitHub version control and validation for XML data and collaborative spreadsheet grids
 
+### Libraries and Tooling
+
+* [Transformers](https://github.com/huggingface/transformers) - Natural Language Processing for TensorFlow 2.0 and PyTorch
+* [Tokenizers](https://github.com/huggingface/tokenizers) - Tokenizers optimized for Research and Production
+
```
Comment (NirantK) [inline README.md]: Err, the bulk of this repo is libraries and tooling. It calls them out as "Libraries"


Please add these to the Python list itself?

File: README.md [hunk 2]
```diff
@@ -88,6 +88,7 @@ Introductions and Guides to NLP
 * [Natural Language Processing: An Introduction](https://academic.oup.com/jamia/article/18/5/544/829676) - Oxford
 * [Deep Learning for NLP with Pytorch](https://pytorch.org/tutorials/beginner/deep_learning_nlp_tutorial.html)
 * [Hands-On NLTK Tutorial](https://github.com/hb20007/hands-on-nltk-tutorial) - NLTK Tutorials, Jupyter notebooks 
+* [Train a new language model from scratch](https://huggingface.co/blog/how-to-train) - Hugging Face 🤗
```
Comment (NirantK) [inline README.md]: Thanks for adding this!

File: README.md (hunk 1 above)
Comment (joshchagani) [reply README.md]: Sure thing!

**Comment by joshchagani** [comment]
I've relocated the links to "Python" in 916d574. Let me know if you need anything else changed!

**Pull request merged**